sklearn/model_selection/_validation.py

```diff
@@ -488,7 +488,14 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
 
     train_scores = {}
     if parameters is not None:
-        estimator.set_params(**parameters)
+        # clone after setting parameters in case any parameters
```

Comment:
I wonder if someone had code relying on the existing behaviour.

Add a test for this wrt cross_validate??

Reply:
Possibly, but I'm not sure what to do about that.
What do you want tested?

Reply:
No, parameters is not set by `cross_validate`. Could add a test for `validation_curve`. But I'm okay without.